components/inputs/input-text.js

```diff
@@ -542,6 +544,12 @@ class InputText extends FocusMixin(LabelledMixin(FormElementMixin(SkeletonMixin(
 		));
 	}
 
+	_handleClick(e) {
+		const input = this.shadowRoot && this.shadowRoot.querySelector('.d2l-input');
```

Comment:
We can use optional chaining now for this kind of thing:
```suggestion
		const input = this.shadowRoot?.querySelector('.d2l-input');
```